Description: hide the "Default" theme
 Under Debian & Ubuntu the Adwaita theme is shown as "Default".
 The intention of upstream code is to hide Adwaita that does not
 render correctly in budgie. This patch hides this theme.
Author: David Mohammed <fossfreedom@ubuntu.com>
Forwarded: not-needed
Last-Update: 2020-04-06

--- a/src/panel/settings/themes.vala
+++ b/src/panel/settings/themes.vala
@@ -50,6 +50,7 @@
 		"Breeze",
 		"Clearlooks",
 		"Crux",
+		"Default",
 		"Emacs",
 		"Industrial",
 		"Mist",
